src/System.IO.Compression/src/System/IO/Compression/DeflateStream.cs

```diff
@@ -209,6 +209,14 @@ public override void Flush()
             return;
```

Comment:
Talking to Eric, he thinks this sounds like a bug (should we be delegating to _stream?)